main.go

```diff
@@ -33,41 +33,32 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
These config options should be moved to the config file and flags could be kept here to override the config from the file.

Reply:
Done, I reintroduced the flags.